ui/src/actions/login.js

```diff
@@ -59,58 +83,31 @@ export function sendSelectProposal(number, navigate) {
   };
 }
 
-export function startSession(userInControl) {
-  return (dispatch) => {
-    dispatch(getInitialState(userInControl));
-    dispatch(setLoading(false));
-  };
-}
-
 export function getLoginInfo() {
-  return (dispatch) =>
-    fetchLoginInfo().then(
-      (loginInfo) => {
-        dispatch(setLoginInfo(loginInfo));
-        return loginInfo;
-      },
-      () => {
-        dispatch(showErrorPanel(true));
-        dispatch(setLoading(false));
-      },
-    );
+  return async (dispatch) => {
+    try {
+      const loginInfo = await fetchLoginInfo();
+      if (!loginInfo.loggedIn) {
+        throw new Error('Not authenticated');
+      }
```

Comment:
As discussed, I think we can remove this `throw` and the `catch` below, since it cannot happen if the app is coded properly. If we get to this and the user is still unauthenticated, then we will hit a 401 when fetching other things later as expected (e.g. in `getInitialState`).

Reply:
:+1: